code/modules/xenomorph/xeno_structures.dm

```diff
@@ -202,6 +202,7 @@
 	. = ..()
 	ammo = GLOB.ammo_list[/datum/ammo/xeno/acid/heavy/turret]
 	associated_hive = GLOB.hive_datums[hivenumber]
+	obj_integrity = 600
```

Comment:
Put this in the define no need for it to be on initialize
Also wont 2 healing per tick mean it takes
300 seconds to heal to full